src/extension-runners/chromium.js

```diff
@@ -307,6 +307,10 @@ export class ChromiumExtensionRunner {
       })
     );
 
+    if (!this.wss) {
+      throw new Error('WebSocketServer closed');
```

Comment:
What's this doing here? There will be a runtime error at `this.wss.address()` below, anyway.

Removing this would reduce the need for extra test coverage. (but since you have test coverage anyway, perhaps tweak the error message if you care about the scenario)

Reply:
I opted to replace this with a $FlowIgnore suppress comment.